static/include/components/board/test-board-digital-interrupts.md

```diff
@@ -0,0 +1,6 @@
+#### Test `digital interrupts`
```

Comment:
Consider taking headings out of includes so they can be consistent.
Also should this have an underscore?